src/com/leetcode/problems/NumberOf1Bits.java

```diff
@@ -0,0 +1,20 @@
+package com.leetcode.problems;
+
+// https://leetcode.com/problems/number-of-1-bits/
+
+public class NumberOf1Bits {
+  public static void main(String[] args) {
+    System.out.println(hammingWeight(10));
+  }
+
+  public static int hammingWeight(int n) {
+    int count = 0;
+    String str = Integer.toBinaryString(n);
```

Comment:
Can you figure out a solution using Java's bitwise operators that you learned about in OCA?